src/data/nat/basic.lean

```diff
@@ -334,6 +334,12 @@ begin
   exact add_lt_add_of_lt_of_le hab (nat.succ_le_iff.2 hcd)
 end
 
+lemma add_left_injective (a : ℕ) : function.injective (λ x, x + a) :=
```

Comment:
Oh, it seems we already have [`add_left_injective`](https://leanprover-community.github.io/mathlib_docs/find/add_left_injective) and [`add_right_injective`](https://leanprover-community.github.io/mathlib_docs/find/add_right_injective); I didn't realize this when I was reviewing the other PR. 

`nat.mul_left_injective` and `nat.mul_right_injective` are OK though.

Reply:
Oh nice catch!